classes/mail/mailables/UserCreated.php

```diff
@@ -49,9 +49,12 @@ class UserCreated extends Mailable
     ];
     protected static ?string $variablePassword = 'password';
 
-    public function __construct(Context $context)
+    protected ?string $password;
+
+    public function __construct(Context $context, string $password = null)
```

Comment:
Let's make the `$password` argument mandatory and add it when the mailable is instantiated, e.g., in `UserXmlPKPUserFilter::parseUser()`

Reply:
Done .